httpcore/_bytestreams.py

```diff
@@ -0,0 +1,77 @@
+from typing import AsyncIterator, Iterator, Callable
+from ._async.base import AsyncByteStream
+from ._sync.base import SyncByteStream
+
+
+class SimpleByteStream(AsyncByteStream, SyncByteStream):
```

Comment:
What do we think of…?

```suggestion
class PlainByteStream(AsyncByteStream, SyncByteStream):
```

Or maybe just…?

```suggestion
class ByteStream(AsyncByteStream, SyncByteStream):
```

Reply:
Tricky one, but a fair call yup. I've gone with `PlainByteStream`.